src/SuperDumpService/Views/Home/_System.cshtml

```diff
@@ -11,34 +11,57 @@
 	<div id="system-info" class="panel-collapse in">
 		<div class="panel-body">
 			<dl class="dl-horizontal">
-				<dt>OS version:</dt>
-				<dd>@Model.Result.SystemContext.OSVersion</dd>
+				@if (Model.Result.SystemContext.GetType() == typeof(SuperDump.Models.SDCDSystemContext))
```

Comment:
you could use `Model.Result.SystemContext is SuperDump.Models.SDCDSystemContext`